src/main.ts

```diff
@@ -2,77 +2,108 @@ import "./style.css";
 import * as wn from "webnative";
 import FileSystem from "webnative/fs/index";
 
-const state = await wn
-  .initialise({
-    permissions: {
-      app: {
-        name: "mumblr",
-        creator: "Jess Martin",
-      },
-      fs: {
-        private: [wn.path.directory("Posts")], // This will be `private/Posts`
-        public: [wn.path.directory("Posts")], // This will be `public/Posts`
+// Check if the user is logged in.
+// TODO: Figure out how to do this!
+
+// If user is _not_ logged in, hide the interface.
+//   Currently, it starts out hidden. Nothing to do here.
+
+// If user is logged in, show the log in button.
+//   Currently, it starts out shown. Nothing to do here.
+
+async function main() {
```

Comment:
I am thinking we want this main function to return the results of wn.initialise() in order to fix the flickering when  redirecting to fission lobby

Reply:
I think you're right!